### PR TITLE
[ci] Use transfer.sh to share logs

### PR DESCRIPTION
**Pull request by hennevogel**
https://transfer.sh/ is a free, public, awesome service to share files

### CODE REVIEW

**Comment by hennevogel** [comment]
Here is an [example build](https://travis-ci.org/openSUSE/open-build-service/jobs/343944710). Scroll down and expand the `after_failure` section.

screenshot from 2018-02-21 10-38-38

File: dist/ci/travis_after_failure.sh [hunk 1]
```diff
@@ -1,30 +1,12 @@
...
-stringToSign="PUT\n\ntext/plain\n${dateValue}\n/obs-travis-articafts/$TRAVIS_BUILD_NUMBER/$TRAVIS_JOB_NUMBER/$1"
-signature=`echo -en ${stringToSign} | openssl sha1 -hmac ${ARTIFACTS_SECRET} -binary | base64`
-curl -X PUT -T $1 \
-  -H "Host: obs-travis-articafts.s3.amazonaws.com" \
-  -H "Date: ${dateValue}" \
-  -H "Content-Type: text/plain" \
-  -H "Authorization: AWS ${ARTIFACTS_KEY}:${signature}" \
-  https://obs-travis-articafts.s3.amazonaws.com/$TRAVIS_BUILD_NUMBER/$TRAVIS_JOB_NUMBER/$1
+function upload_files {
+  for file in *; do
```
Comment (ChrisBr) [inline dist/ci/travis_after_failure.sh]: Does this include the capybara images?

File: dist/ci/travis_after_failure.sh [hunk 2]
```diff
@@ -1,30 +1,12 @@
...
+    echo -n "Uploading $file ->   "
+    curl --upload-file ./$file https://transfer.sh/${TRAVIS_BUILD_NUMBER}-${TRAVIS_JOB_NUMBER}_${file}
```
Comment (ChrisBr) [inline dist/ci/travis_after_failure.sh]: Hm, what about having OBS as a prefix?

Comment (hennevogel) [reply dist/ci/travis_after_failure.sh]: There is no listing so it doesn't really matter right?

File: dist/ci/travis_after_failure.sh (hunk 1 above)
Comment (hennevogel) [reply dist/ci/travis_after_failure.sh]: No, as I'm not sure this still works (and how it works)...

File: dist/ci/travis_after_failure.sh (hunk 2 above)
Comment (ChrisBr) [reply dist/ci/travis_after_failure.sh]: More regarding potential clashing names...

Comment (hennevogel) [reply dist/ci/travis_after_failure.sh]: Can't clash, the append a random directory :-)

**Comment by hennevogel** [comment]
codecov hangs, merging anyway...

**Pull request merged**